### DISCUSSION TITLE
Compass direction in map rotation

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#688] [Q&A] [answered]
[Kurviger 1 Pro & Cruiser 5.1](https://youtube.com/shorts/SQ8-9k_uCnI?si=VVu_UMH98jqAgauz)

**Comment by GuG1050** [comment]
???

**Reply by 0709wiwiwi** [reply]
Where is north ? 
- Check and compare the compass view.
  the red arrow does not point to the north and the arrow rotation seems to be wrong ?

**Comment by devemux86** [comment]
Better use the navigators as they are expected to be used with full screen.
Not sharing device's screen and limited resources with another navigator.

**Reply by devemux86** [reply]
Next time please report with details about what we are supposed to see.
Also provide the route and app profile so we can try to reproduce the report.

> the red arrow does not point to the north and the arrow rotation seems to be wrong ?

I will check it.

**Comment by devemux86** [comment, marked as answer]
- Compass directions is fixed in https://github.com/devemux86/cruiser/discussions/684
